build.sbt

```diff
@@ -3302,7 +3303,17 @@ lazy val `common-polyglot-core-utils` = project
     libraryDependencies ++= Seq(
       "com.ibm.icu"          % "icu4j"    % icuVersion,
       "org.graalvm.polyglot" % "polyglot" % graalMavenPackagesVersion % "provided"
-    )
+    ),
+    modulePath := {
```

Comment:
Thanks you @Akirathan!

To turn a project into a module:
1. `enablePlugins(JPMSPlugin)`
2. define `modulePath` attribute with this magic

Reply:
> What does it mean filterModulesFromUpdate anyway? Which update? What does Compile / update even mean?

`Compile / update` return an [UpdateReport](https://github.com/sbt/librarymanagement/blob/8d71f66997c356790fc887b17e33c7be66a4e590/core/src/main/contraband-scala/sbt/librarymanagement/UpdateReport.scala) that contains all the information about dependency resolution. `filterModulesFromUpdate` tries to pick the absolute path to the modules that are inside this report. This is the easiest way to do this functionality in sbt that I have been able to find.

> Why do we have to specify the module again when it is already among libraryDependencies?

Because of some technical limitations of sbt. `libraryDependencies` is of type `Seq[ModuleID]`. But `modulePath` needs to get `Seq[File]`. And `ModuleID` is nothing more than an Ivy package specification - a mere string. `JPMSUtils.filterModulesFromUpdate` is one way how to get a `File` from `ModuleID`.

> Cannot by default modulePath be derived from class path by filtering all JARs that have module-info.class?

In theory yes. I guess everything is possible with sbt. But in practice, I have no idea how to pull that off with sbt. I was hoping that in this stage, when we are defining more and more JPMS modules ourselves, we would be at least partially migrated to Bazel and that would help with it a bit.

Reply:
Further progress tracked in https://github.com/enso-org/enso/issues/10726